### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by curioussavage**
:eyeglasses: @ajacksified

### CODE REVIEW

File: test/views/layouts/BodyLayout.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  var result;
+
+  before('render and find element', () => {
+
+    var props = {};
+    props.data = new Map();
+    props.dataCache = {};
+    props.ctx = {};
+    props.app = {};
+    props.children='hello world';
```
Comment (ajacksified) [inline test/views/layouts/BodyLayout.jsx]: :no_bell: might as well
```
props = {
  data: new Map(),
  dataCache: {},
```

etc. Could even create a test helper for default props.

**Comment by ajacksified** [comment]
:no_bell: nit, but

:fish:

**Comment by curioussavage** [comment]
:haircut:

**Comment by curioussavage** [comment]
also :eyeglasses: @madbook. 

added a renderWithProps helper that returns the component instance (for setting state or calling methods), the renderer (to get the updated results of render), and the result of the first render.

**Comment by ajacksified** [comment]
:fish:

**Pull request closed without merging**